packages/webapp/pages/squads/[handle]/create.tsx

```diff
@@ -67,6 +67,7 @@ function CreatePost(): ReactElement {
       <WritePage
         onSubmitForm={onClickSubmit}
         isLoading={!isReady || !isDraftReady}
+        isPosting={isPosting}
```

Comment:
it think this should be `isPosting || isSuccess` because for a brief second until page navigation happens "Post" button will become active again and if user clicks on it by mistake we will have double post. Same can be applied to `edit` page.